discovery-provider/src/tasks/users.py

```diff
@@ -338,80 +339,102 @@ def parse_user_event(
     if event_type == user_event_types_lookup["update_multihash"]:
         # Look up metadata multihash in IPFS and override with metadata fields
         if ipfs_metadata:
-            # Fields also stored on chain
```

Comment:
we may want to skip legacy user indexing if the ID generated is past the offset. same for track, playlist already does this.

Reply:
I don't think this should matter as much for now, but am in favor of doing as a fast follow